master/buildbot/www/hooks/github.py

```diff
@@ -170,6 +170,41 @@ def handle_pull_request(self, payload, event):
             len(changes), number))
         return changes, 'git'
 
+    def get_changes(self, category, payload):
```

Comment:
Please call this something like

_get_changes_for_misc_event()

add a comment telling that this is not really a change, as it does not reflect a strict code change, but still is a useful event to start a build on.

This needs more explaination also in the doc (it was really not clear for me at first)